assets/js/meta.js

```diff
@@ -1,14 +1,14 @@
 // ==UserScript==
-// @name Autofill Tool - 1.1.8
+// @name Autofill Tool - 1.1.9
 // @namespace www.cobaltgroup.com
-// @version 1.1.8
+// @version 1.1.9
 // @run-at document-end
 // @downloadURL https://github.com/cirept/autofillReplacer/raw/master/assets/js/meta.user.js
 // @resource toolStyles https://cdn.rawgit.com/cirept/autofillReplacer/1.1.8/assets/css/autofill.css
 // @resource changeLog https://cdn.rawgit.com/cirept/autofillReplacer/1.1.8/docs/LatestChanges.md
 // @resource toolInstructions https://cdn.rawgit.com/cirept/autofillReplacer/1.1.8/docs/README.md
-// @include http://websites.cobalt.com/wsm/editSite.do*
-// @include http://websites.cobalt.com/wsm/cms.do*
+// @include https://websites.cobalt.com/wsm/editSite.do*
```

Comment:
include the http version of the url to cover any issues with someone using the http version of the site.

Reply:
I added the http version and pushed a commit.